src/main/java/com/appian/intellij/k/settings/KSettings.java

```diff
@@ -0,0 +1,30 @@
+package com.appian.intellij.k.settings;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class KSettings {
+  private List<String> internalPrefixes = Collections.singletonList(".i.");
```

Comment:
For backwards compatibility use .i. as default "internal" prefix

Reply:
Hmm, only now noticed that the original check was for startsWith("i.") or contains(".i.")
I guess the gui can be extended to have both prefixes and substrings for internal names. 
Please let me know if that's what you prefer - I feel like prefixes should be enough, but if this will break conventions for your projects, I could extend the gui to have sections for both startsWith and contains...